ui/src/components/Map/MarkerCluster.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <div style="display: none">
+    <slot v-if="ready"></slot>
+  </div>
+</template>
+
+<script>
```

Comment:
Would we be able to use `<script setup lang=ts>` here to keep consistent with the new component syntax?
Some minor refactoring will need to take place after (removing the export default / setup functions, etc)